packages/table/src/Row/HeaderRow/HeaderRow.stories.tsx

```diff
@@ -20,7 +20,7 @@ const meta: StoryMetaType<typeof HeaderRow> = {
     generate: {
       combineArgs: {
         darkMode: [false, true],
-        disabled: [false, true],
+        // TODO: disabled: [false, true],
```

Comment:
Does anyone know why this was commented out?

Reply:
@spark33

Reply:
What do you mean? It doesn't look like it's commented out in main?